src/app/store/test-types/selectors/test-types.selectors.ts

```diff
@@ -25,12 +27,20 @@ export const selectTestTypesTotal = createSelector(testTypesFeatureState, (state
 
 export const selectTestTypesLoadingState = createSelector(testTypesFeatureState, (state) => state.loading);
 
-export const selectTestTypesByVehicleType = createSelector(selectAllTestTypes, toEditOrNotToEdit, (testTypes, testResult) => {
-  if (testResult) {
-    return filterTestTypes(testTypes, testResult);
-  }
-  return [];
-});
+// eslint-disable-next-line max-len
+export const selectTestTypesByVehicleType = createSelector(
+  selectAllTestTypes,
+  toEditOrNotToEdit,
+  selectTechRecordHistory,
+  (testTypes, testResult, selectRecordHistory) => {
```

Comment:
can we change the name from selectRecordHistory to techRecordHistory

Reply:
definitely, I have changed it now